gateway/event_system.go

```diff
@@ -59,24 +56,11 @@ const (
 	EventTokenDeleted = event.TokenDeleted
 )
 
-// EventMetaDefault is a standard embedded struct to be used with custom event metadata types, gives an interface for
-// easily extending event metadata objects
-type EventMetaDefault struct {
```

Comment:
Once we start moving event structs out, why not move as many or all in the group?